dessia_common/workflow/core.py

```diff
@@ -1039,6 +1014,9 @@ def variable_indices(self, variable: Variable) -> Optional[Union[Tuple[int, int,
             return self.nonblock_variables.index(upstream_variable)
         raise WorkflowError(f"Something is wrong with variable {variable.name}")
 
+    def is_variable_nbv(self, variable: Variable) -> bool:
+        return isinstance(self.variable_indices(variable), int)
```

Comment:
Are you sure this test is enough ?

Reply:
Yes, that is totally sufficient.
Actually, NBVs are some kind of weird objects, between variables and blocks.
They are variables, but variables should be attached to blocks. In this case, we can give it an adress that is `(i_block, i_side, i_port)`. If it isn't attached to a block, well, it kinda behaves like a block and only have an int as an "adress". That is why we can spot it just by testing it is an instance of int.

Any other kind of value here will be faulty anyway